bin/bundle-platform

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
Depends on your requirements? One possibility is a `scripts/` directory, but I haven't figured out how all the pieces work so I don't know if that would actually work.